project-base/docker/php-fpm/docker-php-consumer-entrypoint

```diff
@@ -28,4 +28,4 @@ if [[ ${n} = ${default_tries_before_fail} ]]; then
     exit 1
 fi
 
-php bin/console rabbitmq:consumer -m ${messages_to_consume_count} ${consumer_name} && break
```

Comment:
I do not understand this change, could you please explain it to me?

Reply:
In development and tests there is 1000 runs of consumer so code base is always actual and there are no weird situations during development.

Reply:
ok, thanks :+1: I was just asking about the removal of `&& break` but now I see it was just useless there, right?

Reply:
Right